manifests/00-custom-resource-definition.yaml

```diff
@@ -10,1028 +11,1024 @@ spec:
     listKind: IngressControllerList
     plural: ingresscontrollers
     singular: ingresscontroller
-  preserveUnknownFields: false
-  scope: ""
-  subresources:
-    scale:
-      labelSelectorPath: .status.selector
-      specReplicasPath: .spec.replicas
-      statusReplicasPath: .status.availableReplicas
```

Comment:
Where did this go?

Reply:
It seems it was removed at https://github.com/openshift/api/pull/720

Reply:
tagged the PR.

Reply:
bumped openshift/api again